tox.ini

```diff
@@ -14,9 +14,9 @@ setenv =
     RSTUF_LOCAL_STORAGE_BACKEND_PATH = ./data-test/s
     RSTUF_KEYVAULT_BACKEND = LocalKeyVault
     RSTUF_LOCAL_KEYVAULT_PATH = ./data-test/kv
-    RSTUF_LOCAL_KEYVAULT_ONLINE_KEY_NAME = ./data-test/kv/online_key
-    RSTUF_LOCAL_KEYVAULT_ONLINE_KEY_PASSWORD = password
-    RSTUF_LOCAL_KEYVAULT_ONLINE_KEY_TYPE = ed25519
+    RSTUF_LOCAL_KEYVAULT_KEY_NAME = ./data-test/kv/online_key
+    RSTUF_LOCAL_KEYVAULT_KEY_PASSWORD = password
+    RSTUF_LOCAL_KEYVAULT_KEY_TYPE = ed25519
```

Comment:
We should remove this one (to test as optional).
It will fail all tests because the `app.py` initialization will fail. Besides we defined the parameters as optional the `LocalKeyVault`, it misses the condition in `refresh_settings` (you can blame me for this) to not fetch the configuration from environment variable/Dynaconf.

The required changes are:

1) Storage backend

From:
https://github.com/vmware/repository-service-tuf-worker/blob/cc13a730760f8dc131539771d6e2854432ca0d78/repository_service_tuf_worker/repository.py#L149-L152

To:

```python
 storage_kwargs = { 
     s.argument: settings.store[s.name] 
     for s in settings.STORAGE_BACKEND.settings() 
     if settings.store.get(s.name)
 } 
```

2) Keyvault Backend


From:

https://github.com/vmware/repository-service-tuf-worker/blob/cc13a730760f8dc131539771d6e2854432ca0d78/repository_service_tuf_worker/repository.py#L192-L195

To:
 ```python
keyvault_kwargs = { 
     s.argument: settings.store[s.name] 
     for s in settings.KEYVAULT_BACKEND.settings()
     if settings.store.get(s.name)
 } 
```